src/modm/platform/core/cortex/linker.macros

```diff
@@ -72,24 +72,24 @@ TOTAL_STACK_SIZE   = MAIN_STACK_SIZE + PROCESS_STACK_SIZE;
 %% endmacro
 
 
-%% macro section_stack(memory, start=None)
-	/* Main stack in {{memory}} */
+%% macro section_stack(memory, start=None, suffix="")
+	/* Main stack{{suffix}} in {{memory}} */
 	.stack (NOLOAD) :
```

Comment:
need different name for core1, with some produced large overlapped  segment
`Memory segment 20040000->20041c00 is outside of valid address range for device`

Reply:
```
.stack          0x0000000020040000      0xc00
                0x0000000020040000                __stack1_start = .
.stack          0x0000000020041000      0xc00
                0x0000000020041000                __stack_start = .
```

Reply:
```
%% macro section_stack(memory, start=None, suffix="")
        /* Main stack{{suffix}} in {{memory}} */
-       .stack (NOLOAD) :
+       .stack{{suffix}} (NOLOAD) :
        {
        %% if start != None
```

Reply:
Ooops, interesting that the linker didn't complain!